### DISCUSSION TITLE
Nextcloud AIO doesn't set 'default_phone_region'

### DISCUSSION THREAD

**Discussion by nuschpl** [nextcloud/all-in-one#1780] [Questions] [answered]
### Steps to reproduce
1. Install Nexcloud All-in-One
2. Set all available configuration available via UI
3. In the overwiev configuration issues within admin panel get informationGet warning that you need to manually set 'default_phone_region' 

### Expected behavior 
Have the possibility to set the option as part of install process or withing the admin panel

### Actual behavior 
User need to be aware about AIO architecture and probably manually change on of the containers configuration. The panel points user to installation instructions while setting this specific variable is not part of the process.

### Host OS 
not relevant

#### Nextcloud AIO version 
[Nextcloud Hub 3](https://nextcloud.com/) (25.0.2)

**Comment by szaimen** [comment, marked as answer]
There should be a UI optionn in server that allows to set this directly - nothing we will build into AIO.

However see this: https://github.com/nextcloud/all-in-one#how-to-resolve-security--setup-warnings-displays-the-missing-default-phone-region-after-initial-install

**Reply by ngdangtu-vn** [reply]
Instead of UI, I prefer it to be done in `compose.yml` file. We have to modify the `compose.yml` anyway, why would we have to put configurating steps all over the places right?

**Comment by nuschpl** [comment]
My concern was that AIO should provide deployment which is secure and production ready(without warnings like this) . While this  indeed comes from default configuration of one of the servers the reason one choses AIO instead of manual deployment is to get hassle free experience. So IMHO this should be bug against AIO open until upstream bug (the server you've mentioned) will be solved.

**Reply by szaimen** [reply]
Sorry but keeping feature requests open that are not resolved in the upstream project will not scale. Also it is farely trivial to run this one command imho.

For the server issue, see e.g. https://github.com/nextcloud/server/issues/32613

**Comment by xhente** [comment]
I just did a Nextcloud AIO install and I get the same error. Is this problem still in the same state? Is it necessary to run the command manually in CLI? 
`sudo docker exec --user www-data nextcloud-aio-nextcloud php occ config:system:set default_phone_region --value="yourvalue"`

**Reply by szaimen** [reply]
> Is it necessary to run the command manually in CLI?

Yes

**Comment by dg1sek** [comment]
It is a pity that this the only thing in the whole Nextcloud AIO setup that requires me to use google, find this forum, and then use the above sudo docker exec ... command - as others said previously could that not be done in a UI ?
